Typings are failing

The typings have been failing on your most recent build, preventing newer versions from being used in typescript projects.

```
node_modules/@godaddy/terminus/typings/index.d.ts:11:5 - error TS2411: Property 'verbatim' of type 'boolean | undefined' is not assignable to string index type 'boolean | HealthCheck'.

11     verbatim?: boolean
       ~~~~~~~~

node_modules/@godaddy/terminus/typings/index.d.ts:12:5 - error TS2411: Property '__unsafeExposeStackTraces' of type 'boolean | undefined' is not assignable to string index type 'boolean | HealthCheck'.

12     __unsafeExposeStackTraces?: boolean;
       ~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -1,5 +1,4 @@
 declare module "@godaddy/terminus" {
-
   export type HealthCheck = () => Promise<any>;
 
   export class HealthCheckError extends Error {
@@ -8,10 +7,10 @@ declare module "@godaddy/terminus" {
   }
 
   export type HealthCheckMap = {
-    verbatim?: boolean
+    verbatim?: boolean;
     __unsafeExposeStackTraces?: boolean;
-    [key: string]: HealthCheck | boolean;
-  }
+    [key: string]: HealthCheck | boolean | undefined;
+  };
 
   export interface TerminusOptions {
     healthChecks?: HealthCheckMap;

